docs/fio.md

```diff
@@ -0,0 +1,263 @@
+User for Ansible to log in as. Must authenticate with PUBLIC_KEY/PRIVATE_KEY.
+
+### WORKLOAD_IMAGE
+
+Default: `quay.io/openshift-scale/scale-ci-workload`  
+Container image that runs the workload script.
+
+### WORKLOAD_JOB_NODE_SELECTOR
+
+Default: `false`  
+Enables/disables the node selector that places the workload job on the `workload` node.
+
+### WORKLOAD_JOB_TAINT
+
+Default: `false`  
+Enables/disables the toleration on the workload job to permit the `workload` taint.
+
+### WORKLOAD_JOB_PRIVILEGED
+
+Default: `false`  
+Enables/disables running the workload pod as privileged.
+
+### KUBECONFIG_FILE
+
+Default: `~/.kube/config`  
+Location of kubeconfig on orchestration host.
+
+### PBENCH_INSTRUMENTATION
+
+Default: `false`  
+Enables/disables running the workload wrapped by pbench-user-benchmark. When enabled, pbench agents can then be enabled (`ENABLE_PBENCH_AGENTS`) for further instrumentation data and pbench-copy-results can be enabled (`ENABLE_PBENCH_COPY`) to export captured data for further analysis.
+
+### ENABLE_PBENCH_AGENTS
+
+Default: `false`  
+Enables/disables the collection of pbench data on the pbench agent Pods. These Pods are deployed by the tooling playbook
+and if this option is enabled then tooling playbook needs to be executed prior to this test.
+
+### ENABLE_PBENCH_COPY
+
+Default: `true`  
+
+Enables/disables the copying of pbench data to a remote results server for further analysis.
+As of now, this test requires valid `PBENCH_SERVER` where it will copy results at end of test. 
+
+### PBENCH_SSH_PRIVATE_KEY_FILE
+
+Default: `~/.ssh/id_rsa`  
+Location of ssh private key to authenticate to the pbench results server.
+
+### PBENCH_SSH_PUBLIC_KEY_FILE
+
+Default: `~/.ssh/id_rsa.pub`  
+Location of the ssh public key to authenticate to the pbench results server.
+
+### PBENCH_SERVER
+
+Default: There is no public default.  
+DNS address of the pbench results server.
+
+### SCALE_CI_RESULTS_TOKEN
+
+Default: There is no public default.  
+Future use for pbench and prometheus scraper to place results into git repo that holds results data.
+
+### JOB_COMPLETION_POLL_ATTEMPTS
+
+Default: `10000`  
+
+Number of retries for Ansible to poll if the workload job has completed. 
+Poll attempts delay 10s between polls with some additional time taken for each polling action depending on the orchestration host setup.
+FIO I/O test for many pods and big file sizes can run for hours and either we rise `JOB_COMPLETION_POLL_ATTEMPTS` to
+higt value, or remove fully checking for `JOB_COMPLETION_POLL_ATTEMPTS` for FIO I/O test. 
+
+
+### FIOTEST_PREFIX
+
+Default: `fiotest`
+
+### FIOTEST_CLEANUP
+
+Default: `false`
+This parameter for this test must be `false`. FIO I/O test consist of two steps. In first step fio pods will be created
+and then in second step fio load will be executed against fio pods, so it is important that clusterloader which is used
+to create pods does not delete pods after fist part of test is finished. 
```

Comment:
I think the description here is incorrect.  Cleanup is not used in cluster loader in this test and just how it is described here if it was, then the fio pods would immediately be destroyed post being created.  Rather this variable determines if after the test if the pods will remain on the cluster or be cleaned up.  In that case my vote would be to have the default be `true` such that the cluster should return as closely as possible to before the test (Thus another test can be executed from the CI).

Reply:
this can be questionable - for sake of merging this changed to `true`